Source/Events.Processing/Streams/Partitioned/ScopedStreamProcessor.cs

```diff
@@ -54,7 +54,7 @@ protected override async Task<IStreamProcessorState> ProcessEvent(StreamEvent @e
             var streamProcessorState = currentState as StreamProcessorState;
             if (streamProcessorState.FailingPartitions.Keys.Contains(@event.Partition))
             {
-                var newState = new StreamProcessorState(@event.Position + 1, streamProcessorState.FailingPartitions);
+                var newState = new StreamProcessorState(@event.Position + 1, streamProcessorState.FailingPartitions, DateTimeOffset.MinValue);
```

Comment:
The same as the above comment; This resets the LastSuccessfullyProcessed value to the minimum. It should be the last value streamProcessorState.LastSuccessfullyProcessed